ra-optimisation/rao-commons/src/main/java/com/farao_community/farao/rao_commons/linear_optimisation/iterating_linear_optimizer/IteratingLinearOptimizer.java

```diff
@@ -83,6 +83,8 @@ public String optimize(RaoData raoData) {
         String optimizedVariantId;
         for (int iteration = 1; iteration <= parameters.getMaxIterations(); iteration++) {
             optimizedVariantId = cracVariantManager.cloneWorkingVariant();
+            raoData.getCracResultManager().copyCommercialFlowsBetweenVariants(cracVariantManager.getWorkingVariantId(), optimizedVariantId);
```

Comment:
could we copy the results here if and only if the loopFlowApproximationLevel is FIXED_PTDF or UPDATE_PTDF_WITH_TOPO ?

Reply:
And could we do this in the runSensitivityAndUpdateResults() function instead ? I think it would be more readable